docs/src/components/ComponentDoc/ComponentExamples.tsx

```diff
@@ -30,23 +30,22 @@ export default class ComponentExamples extends React.Component<ComponentExamples
    */
   private renderExamples = (): JSX.Element | null => {
     const { displayName } = this.props
-    const allPaths = exampleContext.keys()
 
     // rule #1
-    const indexPath = _.find(allPaths, path =>
+    const indexPath = _.find(exampleGroupsContext.keys(), path =>
```

Comment:
while `Groups` is correct, maybe call it `exampleIndexContext`, it will be easier to infer which files it corresponds to. For example, it has take me to check sources of this `exampleGroupsContext` to ensure that `Groups` implies index files only (the thing that is expected by `indexPath ` variable)

Reply:
Was done in #644.